File: TP/TP1/Code/4C.cpp
/* Saisir une valeur entre deux bornes choisies par l’utilisateur */
#include <iostream>

int main()
{
  int a;

  do {
    std::cout << "Donner une valeur entre 1 et 10 : " << std::flush;
    std::cin >> a;
  } while ((a < 0) || (a > 10));

  std::cout << "Tout est OK !" << std::endl;
  
  return 0;
}

File: TP/TP1/Code/Python/4C.py
#!/usr/bin/python
# Saisir une valeur entre deux bornes choisies par l’utilisateur
print('Donne moi une valeur : ', end='')
foo = int(input())

while foo < 0 or foo > 20:
    print('La valeur doit être entre 0 et 20. Recommence : ', end='')
    foo = int(input())
